packages/integration-tests/src/client/index.ts

```diff
@@ -22,7 +22,7 @@ export default class MockClient {
   private readonly storage: MemoryStorage;
   private readonly logto: LogtoClient;
 
-  constructor(config?: LogtoConfig) {
+  constructor(config?: Partial<LogtoConfig>) {
```

Comment:
Why change to `Partial` here?
Does `defaultConfig` cover all required props a client needs?

Reply:
Yep